integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDTFAlignByTimeQueryIT.java

```diff
@@ -156,6 +156,36 @@ private static void registerUDF() {
     }
   }
 
+  @Test
+  public void queryWithUnquotedAttributes() {
+    try (Connection connection = EnvFactory.getEnv().getConnection();
+        Statement statement = connection.createStatement()) {
+      try {
+        statement.executeQuery("select sum_sec(s1, 'interval'=3) from root.udf.d1;");
+      } catch (SQLException e) {
+        assertTrue(e.getMessage().contains("Attributes of functions should be quoted"));
+      }
+      try {
+        statement.executeQuery("select sum_sec(s1, interval=3) from root.udf.d1;");
```

Comment:
This SQL is legal in some cases. For example, the second parameter of the function is a sequence of Boolean types.

I think we need to add checks on function signatures.